ecl/eclccserver/eclccserver.cpp

```diff
@@ -339,8 +339,17 @@ class EclccCompileThread : implements IPooledThread, implements IErrorReporter,
         eclccCmd.append(" --timings --xml");
         eclccCmd.append(" --nostdinc");
         eclccCmd.append(" --metacache=");
-        VStringBuffer logfile("%s.eclcc.log", workunit->queryWuid());
+
+#ifdef _CONTAINERIZED
+        eclccCmd.append(" --nologfile");
```

Comment:
Have you checked the output?  the output to stdout is not in the standard logging format - e.g., it doesn't include the id/audience etc.  That should be addressed if the output from eclcc is going to be sent to the standard log processing.

Reply:
@ghalliday - please see new commit.

I have checked in bare-metal / containerized.
In bare-metal logging is captured to a file as before, but moved to /var/log/.. (prev. in /var/lib).
In containerized, logging is output to stdout and seen by K8s.